### DISCUSSION TITLE
How to develop a SuperController for the FAST.FARM.

### DISCUSSION THREAD

**Discussion by Hermse** [OpenFAST/openfast#1307] [Q&A] [answered]
The description of the super controller in the user documentation is very brief.

**Comment by jjonkman** [comment, marked as answer]
Dear @Hermse,

The super controller available in FAST.Farm is described in the online documentation [here](https://openfast.readthedocs.io/en/main/source/user/fast.farm/ModelGuidance.html#super-controller) and [here](https://openfast.readthedocs.io/en/main/source/user/fast.farm/FFarmTheory.html#fast-farm-theory-basis).

A functional super controller example is provided in the following topic on our forum: https://forums.nrel.gov/t/fast-farm/1887/161. Before enabling the super controller, however, I would ensure that FAST.Farm runs as expected without the super controller.

Best regards,

**Reply by Hermse** [reply]
Thank you very much, your reply helped me a lot.
